src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -9,20 +10,24 @@
             [status-im.ui.screens.accounts.recover.styles :as styles]
             [status-im.ui.screens.accounts.recover.db :as recover.db]
             [status-im.ui.screens.accounts.db :as db]
+            [status-im.react-native.js-dependencies :as js-dependencies]
             [cljs.spec.alpha :as spec]
             [status-im.ui.components.common.common :as components.common]))
 
 (defview passphrase-input [passphrase]
-  (letsubs [error [:get-in [:accounts/recover :passphrase-error]]]
+  (letsubs [error [:get-in [:accounts/recover :passphrase-error]]
+            input-ref (reagent/atom nil)]
+    {:component-did-mount #(.hideView js-dependencies/testfairy @input-ref)}
     [text-input/text-input-with-label
-     {:style          {:flex 1}
-      :height         92
-      :label          (i18n/label :t/passphrase)
-      :placeholder    (i18n/label :t/enter-12-words)
-      :multiline      true
-      :default-value  passphrase
-      :on-change-text #(re-frame/dispatch [:set-in [:accounts/recover :passphrase] %])
-      :error          error}]))
+     {:style               {:flex 1}
```

Comment:
Use components/flex

Reply:
Sorry, but seems like not in scope of this PR, i didn't change that line.

Reply:
Right but that one of those easy win we can make along the line